2022 hybrid updates

RvanB opened the pull request:
All front-end updates, individual webpage changes separated in commits (for the most part).

RvanB committed:
Homepage text changes, reorder 7 Ws for better spacing and small screen order

RvanB committed:
Consistently use Roboto as main font

Semantic UI had Lato set as its page font, which was being used sparsely in some semantic ui elements and looked strange. This small change makes semantic ui use Roboto for all semantic ui elements to be consistent with the rest of the website.

RvanB committed:
Rules of Play and FAQ updates

RvanB committed:
Update line in FAQ

RvanB committed:
Remove postsecondary division

RvanB committed:
Register page updates (hold harmless)

RvanB committed:
Add shirt design to FAQ

RvanB committed:
Gear page updates, new color previews

RvanB committed:
Styling fixes for mobile

RvanB committed:
Update Rules of Play Mechanics

RvanB committed:
Truthful entry register page message

RvanB committed:
Fix duplicate activeIndex

RvanB committed:
Millie's requested changes

cabeese commented on client/components/public/FAQ.jsx:
```diff
@@ -61,48 +63,71 @@ const importantDates = (
+
+const schedule_virtual_data = [
+  {
+    time: "9:30–10:15 AM",
```
Could we keep the `(PT)` on here for the virtual stuff, since people may still be in different time zones?

RvanB replied:
I agree, I believe that was in my notes (that I forgot to bring up). I decided to copy Millie's changes exactly but I'll add back in the PT.

cabeese commented on client/components/public/FAQ.jsx:
```diff
@@ -130,27 +155,26 @@ FAQ = class FAQ extends Component {
-          <Accordion.Content active={activeIndex === 1}>
-            <Header as='h3' icon={<Icon color="blue" name="tag"/>} content="Address" />
+          <Accordion.Content active={activeIndex === 0}>
+            <h2><Icon color="blue" name="tag"/> Address</h2>
```
Any reason we moved away from `<Header as='h2'...` in this case? prob fine if we need to do it manually, just seems out of place

RvanB replied:
Yeah there is a reason for that, for some reason the <Header as='h2'... changes the alignment of the Icon with the actual text in the header.

cabeese commented on client/components/public/FAQ.jsx:
```diff
@@ -278,11 +303,14 @@ FAQ = class FAQ extends Component {
-            </p>
+            <List bulleted>
+              <List.Item>Students, Faculty, Staff, Alumni, Community, Family, Everyone, Anywhere!</List.Item>
+              <List.Item>In the interest of safety, the in-person option this year is only open to the WWU campus community this year.</List.Item>
```
nit: "this year" twice in this sentence

RvanB replied:
Good catch!

cabeese commented on client/components/register/imports/RegisterForm.jsx:
```diff
@@ -348,14 +350,23 @@ class RegisterForm extends Component {
-    return (
-      <Segment basic>
-        <p>I hereby acknowledge that I have voluntarily chosen (or voluntarily chosen to allow my minor child) to participate in the {eventYear} WWU Great Puzzle Hunt sponsored by the WWU Mathematics Department, held on {eventDay}, {eventDate}. I understand the risks involved in the Puzzle Hunt, including the unlikely but potential risk of injury to me (or my minor child), and I agree to accept a ...
+    {/*<Segment basic>
```
Why is this commented out and then re-added below..? If we don't need this old hold harmless, can we just delete it?

RvanB replied:
It just seems like we often revert to older versions of text from when there was an in-person only event, I'm happy to delete it if we think it won't be used though.

cabeese commented on client/components/public/Home.jsx:
```diff
@@ -29,12 +29,12 @@ const AnnouncementsMessage = (
                       </ul>
                       <br />
-                      <center><strong>Check back in November for updates!</strong></center>
+                      <center><strong>Check back here for updates!</strong></center>
```
Could we add a note with today's date saying we're delaying registration by a little longer to implement some technical upgrades?

RvanB replied:
Will do! After all we did say there would be updates.

cabeese reviewed (approved):
Looks good! Couple small changes I'd like to see but I think it's in good shape otherwise

RvanB committed:
Requested changes

-Add timezone to virtual event schedule
-Remove redundant "this year"
-Remove commented hold harmless
-Add update to homepage banner
